.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -9,8 +9,8 @@ body:
         *Before reporting:*
         - Confirm the problem is reproducible on [**master**](https://github.com/neovim/neovim/releases/nightly) or [**latest stable**](https://github.com/neovim/neovim/releases/stable) release
         - run `make distclean` when encountering build issues
-        - search [existing issues](https://github.com/neovim/neovim/issues?q=is%3Aissue+is%3Aopen+label%3Abug)
-        - check the [FAQ](https://github.com/neovim/neovim/wiki/FAQ)
+        - search [existing issues](https://github.com/neovim/neovim/issues?q=is%3Aissue+is%3Aopen+label%3Abug,bug-crash) (including [closed!](https://github.com/neovim/neovim/issues?q=is%3Aissue+is%3Aclosed+label%3Abug%2Cbug-crash))
```

Comment:
```suggestion
        - search [existing issues](https://github.com/neovim/neovim/issues?q=is%3Aissue+is%3Aopen+label%3Abug,bug-crash) (including [closed](https://github.com/neovim/neovim/issues?q=is%3Aissue+is%3Aclosed+label%3Abug%2Cbug-crash))
```

Reply:
Not specific to this PR but it gave me the idea: what do we think about making each of these items a checkbox? That gives us (the triagers) some feedback that users actually have read the items (it's true that some people will blindly click every checkbox anyway, but I predict _most_ won't).

So e.g. something like this:

```
*Before reporting:*
- [ ] I have confirmed the problem is reproducible on **master** or **latest stable**
- [ ] If my issue is a build issue, I have run `make distclean`
- [ ] I have searched for existing issues
```

etc.

Reply:
Hmm, not sure I'm in favor of this, for three reasons:
1. as you mentioned many (more than you expect; I wager most) people will just ignore this (or blindly click if we enforce this; see next point);
2. it's hard to enforce -- what will we do when something isn't ticked? close the issue? politely ask them (as we do now)?
3. such checkboxes appear as a task list in the PR view; this can be helpful (only need to look at PRs where all tasks are completed) but will make _actual_ task lists in tracking-type issues less visible.

Such boxes would be much more valuable if they would actually guide the user through steps (kinda like your second example) -- so less "I _swear_ I did X" and more "First do X, then..."

But this would require the steps to be relevant to the issue, and we'd end up creating LunarVim-startup-flowcharts...